src/extension.ts

```diff
@@ -357,15 +357,9 @@ const runValeOnWorkspace = async (): Promise<ValeDiagnostics> => {
     // Explicitly find all elligible files ourselves so that we respect
     // "files.exclude", ie, only look at files that are included in the
     // workspace.
-    const extensions: ReadonlyArray<string> = [
-        "md",
-        "markdown",
-        "txt",
-        "rst",
-        "tex",
-        "adoc",
-        "asciidoc",
-    ];
+
+    const extensions: Array<string> = workspace.getConfiguration("vscode-vale").get<string[]>("fileExtensions")!;
```

Comment:
Nitpick: to let TS infer the return value automatically

```suggestion
    const extensions = workspace
      .getConfiguration("vscode-vale")
      .get<ReadonlyArray<string>>("fileExtensions")!;
```

<img width="439" alt="image" src="https://user-images.githubusercontent.com/1110551/69611688-a0c49800-102e-11ea-9ad3-ffbede3179f8.png">